README.md

```diff
@@ -76,20 +76,22 @@ long_line_under_100_chars
 # Extra Specs Coding Style
 
 - Add a blank line between `let` and `before`.
+- Prefer `ModelKlass.new` over `build_stubbed` over `build` over `create` for optimal performance.
 - Use the following as a rule of thumb for context setup. If you can't follow it, stop for a second and think about your object's API, maybe the problem is there.
 
 ```ruby
-# after describe the first is always the subject
+# after describe the first is always the subject, called the same as the described method.
```

Comment:
`the subject, called the same as the described method` is still a no go for me, especially for readability.
Sometimes you're testing the method process and sometimes you're testing the returned object. 
Like
```
expect { merge }. to raise Something
```
and
```
expect(merged_object).to contain something
```

The naming convention is quite dependent of where it's implemented. I don't think it's a good idea to make the spec looks like it was generated by a bot.

Reply:
Also in some cases calling it after method reduces readability, especially that method alone doesn't sound good. So I'd agree with Marc, use what sounds best in specific case rather than enforce method name always.
for me `expect { call }.to change { Message.count }` sounds worse than `expect { add_message }.to change { Message.count }`

Reply:
Also, there might be some collision in methods' names in global namespace.

Reply:
This is a rule of thumb as it says 2 lines above, start from this and change upon need.
I also change this for controller specs like `get_index patch_update post_create` etc, where I'm not in control of the API.

For `call`:
> If you can't follow it, stop for a second and think about your object's API, maybe the problem is there.

Why don't you call the method `add_message` then? Or `add`? Something more descriptive.

For `merge`:
You're defining the return value of a method here, not writing english.
`expect(merge).to eq ["something"]` is fine.

Reply:
name of the class is descriptive enough in such case and call will allow to use proc as well.
`CreateMessage.call(params)` is perfectly fine

For merge - you do but if it reads like normal language it's much easier to understand.

Reply:
So this is subjective, but still a good rule of thumb.

Reply:
I agree with rule of thumb, but the part `is always the subject, called the same as the described method.` (especially word always) does not allow for any exceptions 😛
So basically - I'm ok with using that as long as we can use common sense and apply wording that makes most sense in specific case

Reply:
Just read 1 line above.

Reply:
:doge: didn't notice it refers to whole section 😄